packages/gatsby-theme-api-docs/src/components/description.js

```diff
@@ -17,7 +17,7 @@ const DescriptionTextContainer = styled.span`
 // as a property. Use this to embed description into a layout like
 // a table, card or bullet list.
 export const DescriptionText = (props) => (
-  <DescriptionTextContainer>
+  <DescriptionTextContainer className="embedded-api-description">
```

Comment:
Let's avoid using classnames as references. We can use e.g. `data-search-key` attributes and use the query selector like `[data-search-key="embedded-api-description"]`